apps/bot/commands/slash/get-answer-count.ts

```diff
@@ -0,0 +1,68 @@
+import { SlashCommandBuilder } from 'discord.js'
+import { SlashCommand } from '../types.js'
+import { getCorrectAnswersCount } from '../../db/actions/users.js'
+import { isUserProfilePublic } from '../../utils.js'
+
+export const command: SlashCommand = {
+  data: new SlashCommandBuilder()
+    .setName('get-answer-count')
+    .setDescription('Get the number of your accepted answers')
+    .addUserOption((user) =>
+      user
+        .setName('user')
+        .setDescription('The user to get number of marked answers for')
+        .setRequired(false),
+    )
+    .setDMPermission(false),
+
+  async execute(interaction) {
+    // Get the user option
+    const userOption = interaction.options.getUser('user')
+
+    // Use the ID of the user option if it's provided, otherwise use the ID of the interaction user
+    const userId = userOption?.id || interaction.user.id
+    const userArgProvided = !!userOption
+    const count = await getCorrectAnswersCount(userId)
```

Comment:
This function call can potentially take a while to finish so we need to tell Discord that we received the command before they timeout us. You can do that with `deferReply`, we already do this in the `give-points` command:

https://github.com/rafaelalmeidatk/nextjs-forum/blob/a9f7f4bcd07ab1d57dcb480c03e6ed483a6e6f69/apps/bot/commands/slash/give-points.ts#L39-L45

Then you need to replace the `interaction.reply` calls to `interaction.editReply` and remove the `ephemeral` property because it is already set by the defer function